lib/index.js

```diff
@@ -267,11 +274,21 @@ export default {
             range = Helpers.generateRange(textEditor, line);
           }
 
+          // avoid duplicated issues
+          if (fingerprints.has(issue.fingerprint)) {
+            return;
+          }
+          fingerprints.add(issue.fingerprint);
+
           linterResults.push({
-            type: 'Warning',
-            text: issue.description,
-            filePath,
-            range,
+            severity: mapSeverity[issue.severity] || 'warning',
+            excerpt: `${issue.engine_name.toUpperCase()}: ${issue.description} [${issue.check_name}]`,
+            description: (issue.content && issue.content.body) ? issue.content.body : undefined,
+            url: encodeURI(`https:www.google.com/#q=${issue.engine_name} ${issue.check_name}`),
```

Comment:
I will remove it then. Generating a customized URL for each reported issue for each engine would be a time-consuming task right now, and CodeClimate seems to do it for us in the detailed explanation of the issue :wink: